alf/environments/suite_carla.py

```diff
@@ -721,7 +722,8 @@ def get_current_time_step(self, current_frame):
         info = OrderedDict(
             success=np.float32(0.0),
             collision=np.float32(0.0),
-            red_light=np.float32(0.0),
+            red_light_run=np.float32(0.0),
+            red_light_met=np.float32(0.0),
```

Comment:
Suggest explaining briefly what the two stands for, and units.

Reply:
Added comments and also changed their names.